### DISCUSSION TITLE
Apache Guacamole Documentation Question

### DISCUSSION THREAD

**Discussion by reggie-mcmurtrey** [authelia/authelia#8098] [Troubleshooting]
Question about client_secret.

The docs for authelia config states:

`        client_secret: '$pbkdf2-sha512$310000$c8p78n7pUMln0jzvd4aK4Q$JNRBzwAo0ek5qKn50cFzzvE9RXV88h1wJn5KGiHrD0YKtZaR/nCb2CJPOsKaPK0hjf.9yHxzQGZziziccp6Yng'  # The digest of 'insecure_secret'.
`
this is a digest of 'insecure_secret', but I don't see 'insecure_secret' used anywhere in the Guacamole config.  I have searched online for guacamole config mentioning the client secret and i'm not having any luck.  what am I missing here?

Thanks.

**Comment by james-d-elliott** [comment]
I suspect it doesn't support it and this wasn't properly documented by me or whoever added it.
